src/ring_theory/ideals.lean

```diff
@@ -228,6 +228,46 @@ def map_mk (I J : ideal α) : ideal I.quotient :=
   smul_mem' := by rintro ⟨c⟩ _ ⟨x, hx, rfl⟩;
     exact ⟨c * x, J.mul_mem_left hx, rfl⟩ }
 
+/-- The preimage of an ideal J under the quotient map `R → R/I`. -/
+def comap_mk (I : ideal α) (j : ideal I.quotient) : ideal α :=
```

Comment:
This was only necessary since I was trying to work directly with the quotients. Now that I proved a more general statement about surjective maps I was able to remove these defintions